Tasks/VsTest/testselectorinvoker.ts

```diff
@@ -76,11 +77,15 @@ export class TestSelectorInvoker {
                 'filter': pathFilters,
                 'userMapFile': tiaConfig.userMapFile ? tiaConfig.userMapFile : '',
                 'testCaseFilterResponseFile': testCaseFilterFile ? testCaseFilterFile : '',
+                'proxyurl': vstestConfig ? vstestConfig.proxyConfiguration.proxyUrl : dtaTestConfig.proxyConfiguration.proxyUrl,
+                'proxyusername': vstestConfig ? vstestConfig.proxyConfiguration.proxyUserName : dtaTestConfig.proxyConfiguration.proxyUserName,
+                'proxypassword': vstestConfig ? vstestConfig.proxyConfiguration.proxyPassword : dtaTestConfig.proxyConfiguration.proxyPassword,
```

Comment:
why this is not common? repeated 3 times

Reply:
Its 3 different properties. Didnt get you